website/generate-jsdoc.js

```diff
@@ -2,15 +2,32 @@ const jsdoc2md = require("jsdoc-to-markdown");
 const fs = require("fs");
 const path = require("path");
 const prettier = require("prettier");
-const publicFiles = [
+const publicClassFiles = [
   "amplitude-client.js",
   "amplitude.js",
   "identify.js",
   "revenue.js",
 ];
+const optionsFile = "options.js";
 const srcDir = path.join(__dirname, "../", "src");
 const outputDir = path.join(__dirname, "docs");
-function generateMarkdown(inputFile) {
+
+function generateOptionsMarkdown(inputFile) {
```

Comment:
rename it to be `generateTypeofMarkdown`? So that it's more generic.

Reply:
fixed